res/database/Default/user/scr0llbaer_constr_roles.xml

```diff
@@ -0,0 +1,343 @@
+		<programmerole guid="roles-popeye-olive" first_name="Liv" last_name="Oilia" first_name_original="Olive" last_name_original="Oyl" gender="2" comment="https://popeye.fandom.com/wiki/Olive_Oyl | also in language files!" />
+		<programmerole guid="roles-popeye-brutus" first_name="Brachius" last_name="" first_name_original="Brutus" last_name_original="" gender="1" comment="https://popeye.fandom.com/wiki/Brutus" />
+		<programmerole guid="roles-popeye-sweepea" first_name="Swea'Pee" last_name="" first_name_original="Swee'Pea" last_name_original="" gender="1" comment="https://popeye.fandom.com/wiki/Swee%27Pea" />
+		<programmerole guid="roles-popeye-wimpy" first_name="J. Washington" last_name="Whinpey" first_name_original="J. Wellington" last_name_original="Wimpy" gender="1" comment="https://popeye.fandom.com/wiki/J._Wellington_Wimpy" />
+		<programmerole guid="roles-popeye-deezil" first_name="Diz" last_name="Oilia" first_name_original="Deezil" last_name_original="Oyl" gender="2" comment="https://popeye.fandom.com/wiki/Deezil_Oyl | also in language files!" />
+		<programmerole guid="roles-popeye-poopdeck" first_name="Poodock" last_name="Papi" first_name_original="Poopdeck" last_name_original="Pappy" gender="1" comment="https://popeye.fandom.com/wiki/Poopdeck_Pappy" />
+
+		<programmerole guid="roles-transformers-transformer" first_name="Tresforger" last_name="" first_name_original="Transformer" last_name_original="" gender="" comment="https://en.wikipedia.org/wiki/Transformers" />
+		<programmerole guid="roles-transformers-autobot" first_name="Mobibot" last_name="" first_name_original="Autobot" last_name_original="" gender="" comment="https://en.wikipedia.org/wiki/Autobot" />
+		<programmerole guid="roles-transformers-decepticon" first_name="Deflekticon" last_name="" first_name_original="Defecticon" last_name_original="" gender="" comment="https://en.wikipedia.org/wiki/Decepticon" />
+		<programmerole guid="roles-transformers-optimus-prime" first_name="Ultimus" last_name="Premio" first_name_original="Optimus" last_name_original="Prime" gender="1" comment="https://en.wikipedia.org/wiki/Optimus_Prime" />
+		<programmerole guid="roles-transformers-megatron" first_name="Metacon" last_name="" first_name_original="Megatron" last_name_original="" gender="1" comment="https://en.wikipedia.org/wiki/Megatron" />
+		<programmerole guid="roles-transformers-starscream" first_name="Starscreech" last_name="" first_name_original="Starscream" last_name_original="" gender="1" comment="https://en.wikipedia.org/wiki/Starscream" />
+
+				<!-- most Smurfs characters also in language files! -->
+		
+		<programmerole guid="roles-smurfs" first_name="Sluempfs" last_name="" first_name_original="Smurfs" last_name_original="" gender="" comment="TODO: need to abuse programmerole datastructure for now. Perhaps there need to be 'terminology' items for such cases eventually" />
```

Comment:
Why add these roles now, if they are not used at all?
Maybe split them into a separate database directory (WIP=work in progress). In this case they would be quickly available but would not be loaded unnecessarily into every game.

Reply:
yeah I as a bit preemptive with the Smurfs while I was at it going through the characters. If you tell me which file I can separate out unused stuff. Can be generally a good idea to have a "WIP" section.

Reply:
Ronny already suggested a separate database repository, which has advantages and drawbacks. A possibility within this repository would be a WIP-database next to Default (we would have to make sure, not to include it in the distribution).

Reply:
I'll keep it like it is for now, because the main effort is managing the 5 different files with these entries. I'll keep it on my radar to make more use of those roles soon-ish.